exporter/azure_monitor/azure_monitor.go

```diff
@@ -37,17 +38,87 @@ var _ trace.Exporter = (*AzureTraceExporter)(nil)
 	@param sd Span data retrieved by opencensus
 */
 func (exporter *AzureTraceExporter) ExportSpan(sd *trace.SpanData) {
-	baseObj := common.BaseObject {IKey : exporter.options.InstrumentationKey}
 	envelope := common.Envelope {
-		BaseObject: baseObj,
+		IKey : exporter.Options.InstrumentationKey,
+		Tags : common.Azure_monitor_contect,

```

Comment:
typo: contect

Reply:
Yes, it should be context. I will update.